.github/workflows/windows-latest.yml

```diff
@@ -0,0 +1,60 @@
+name: windows-latest
+
+on:
+  workflow_dispatch:
```

Comment:
Is there a case where we need `workflow_dispatch`?

Reply:
`workflow_dispatch` lets us build using web GUI click right? Not an error to have it.

Reply:
Yes, I always like having the option to trigger CI manually, which makes it easier to debug CI issues.